File: app/src/main/java/edu/nju/memo/MainActivity.java
package edu.nju.memo;

import android.content.Intent;
import android.os.Bundle;
import android.support.v4.app.FragmentManager;
import android.support.v7.app.AppCompatActivity;
import edu.nju.memo.activities.fragment.RecyclerViewFragment;



public class MainActivity extends AppCompatActivity {

    private static final String TAG = "MainActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        FragmentManager fragmentManager = this.getSupportFragmentManager();
        fragmentManager.beginTransaction().add(R.id.main_layout, new RecyclerViewFragment()).commit();
    }

}

File: app/src/main/java/edu/nju/memo/MainActivity.kt
package edu.nju.memo

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import edu.nju.memo.service.ClipboardListenService

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        startService(Intent(applicationContext, ClipboardListenService::class.java))
    }
}
